training/tlt/pedestrian_detection/extract_kitti_labels.py

```diff
@@ -0,0 +1,54 @@
+import os
```

Comment:
can you rename this file to extract_labels.py? Since it is not extracting the KITTI dataset labels, the filename is a bit confusing.

Reply:
done